OpenMEEG/libs/OpenMEEG/src/assembleSensors.cpp

```diff
@@ -45,142 +45,145 @@ knowledge of the CeCILL-B license and that you accept its terms.
+
+    void
+    assemble_Head2EEG(SparseMatrix& mat,const Geometry& geo,const Matrix& positions ) {
+
+        mat = SparseMatrix(positions.nlin(),(geo.size()-geo.nb_current_barrier_triangles()));
 
         Vect3 current_position;
         Vect3 current_alphas;
         Triangle current_triangle;
         for ( unsigned i = 0; i < positions.nlin(); ++i) {
             for ( unsigned k = 0; k < 3; ++k) {
-                current_position(k) = positions(i, k);
+                current_position(k) = positions(i,k);
             }
             double dist;
             dist_point_geom(current_position,geo,current_alphas,current_triangle,dist);
-            mat(i, current_triangle.s1().index()) = current_alphas(0);
-            mat(i, current_triangle.s2().index()) = current_alphas(1);
-            mat(i, current_triangle.s3().index()) = current_alphas(2);
+            mat(i,current_triangle.s1().index()) = current_alphas(0);
+            mat(i,current_triangle.s2().index()) = current_alphas(1);
+            mat(i,current_triangle.s3().index()) = current_alphas(2);
         }
     }
 
-    Head2EEGMat::Head2EEGMat(const Geometry& geo, const Sensors& electrodes)
-    {
-        assemble_Head2EEG(*this, geo, electrodes.getPositions());
+    Head2EEGMat::Head2EEGMat(const Geometry& geo,const Sensors& electrodes) {
+        assemble_Head2EEG(*this,geo,electrodes.getPositions());
     }
 
     // ECoG positions are reported line by line in the positions Matrix
     // mat is supposed to be filled with zeros
     // mat is the linear application which maps x (the unknown vector in symmetric system) -> v (potential at the ECoG electrodes)
     // difference with Head2EEG is that it interpolates the inner skull layer instead of the scalp layer. 
-    void assemble_Head2ECoG(SparseMatrix& mat, const Geometry& geo, const Matrix& positions, const Interface& i)
-    {
-        mat = SparseMatrix(positions.nlin(), (geo.size()-geo.nb_current_barrier_triangles()));
+
+    void
+    assemble_Head2ECoG(SparseMatrix& mat,const Geometry& geo,const Matrix& positions,const Interface& i) {
+        mat = SparseMatrix(positions.nlin(),(geo.size()-geo.nb_current_barrier_triangles()));
 
         Vect3 current_position;
         Vect3 current_alphas;
         Triangle current_triangle;
-        for ( unsigned it = 0; it < positions.nlin(); ++it) {
-            for ( unsigned k = 0; k < 3; ++k) {
-                current_position(k) = positions(it, k);
-            }
-            dist_point_interface(current_position, i, current_alphas, current_triangle);
-            mat(it, current_triangle.s1().index()) = current_alphas(0);
-            mat(it, current_triangle.s2().index()) = current_alphas(1);
-            mat(it, current_triangle.s3().index()) = current_alphas(2);
+        for (unsigned it = 0; it<positions.nlin(); ++it) {
+            for (unsigned k = 0; k<3; ++k)
+                current_position(k) = positions(it,k);
```

Comment:
same here.

please please @papadop put cosmit changes in dedicated PRs as you make my review harder with many green lines that are totally unrelated to the fix